gap/congruences/conguniv.gi

```diff
@@ -18,14 +18,43 @@ function(S)
   local fam, cong;
   fam := GeneralMappingsFamily(ElementsFamily(FamilyObj(S)),
                                ElementsFamily(FamilyObj(S)));
-  cong := Objectify(NewType(fam, IsUniversalSemigroupCongruence), rec());
+  cong := Objectify(NewType(fam,
+                            IsSemigroupCongruence and IsAttributeStoringRep),
+                    rec());
   SetSource(cong, S);
   SetRange(cong, S);
+  SetIsUniversalSemigroupCongruence(cong, true);
   return cong;
 end);
 
 #
 
+InstallMethod(IsUniversalSemigroupCongruence,
```

Comment:
This could also be installed as an immediate method, given the additional filter `and HasNrEquivalenceClasses`